exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,17 @@
+final class Proverb {
+    private String[] words;
+
+    Proverb(String[] words) {
+        this.words = words;
+    }
+
+    String recite() {
+        if (words.length < 1) return "";
```

Comment:
Please put curly brackets around the body of this if :)

Reply:
Awesome. Thank you so much. I wasn't sure about that if-statement since it was extremely simple, but I do agree it's much more readable and debuggable with curly braces.